doc_src/history.txt

```diff
@@ -15,21 +15,23 @@ history ( -h | --help )
 
 `history` is used to list, search and delete the history of commands used.
 
-The following options are available:
+The following commands are available:
+
+- `-s` or `--search` returns history items matching the search string. If no search string is provided it returns all history items. This is the default operation if no other operation is specified. Entries are ordered newest to oldest.
```

Comment:
We added a mention of "--contains" here on purpose, because people don't tend to look that far.